src/status_im/chat/models/loading.cljs

```diff
@@ -60,10 +63,38 @@
          message-id)))
    statuses))
 
-(fx/defn initialize-chats
-  "Initialize persisted chats on startup"
+(fx/defn update-chats-in-app-db
+  {:events [:chats-list/load-success]}
+  [{:keys [db] :as cofx} chats]
+  (fx/merge cofx
+            {:db (assoc db :chats chats)}
+            (commands/load-commands commands/register)))
+
+(defn- unkeywordize-chat-names
+  [chats-response]
+  (assoc chats-response
+         :chats
+         (into {}
```

Comment:
sorry I didn't catch that earlier reduce-kv will be more efficient here:
`(reduce-kv (fn [acc chat-name chat] (assoc acc chat-name chat)) {} (:chats chats-response))`

Reply:
I also found a bug there, so thanks :) fixed.